src/content/concepts/hot-module-replacement.md

```diff
@@ -13,7 +14,7 @@ Hot Module Replacement (HMR) exchanges, adds, or removes [modules](/concepts/mod
 
 - Retain application state which is lost during a full reload.
 - Save valuable development time by only updating what's changed.
-- Tweak styling faster -- almost comparable to changing styles in the browser's debugger.
+- Live CSS modification -- almost comparable to changing CSS in the browser's debugger.
```

Comment:
What about?:

> CSS and JS source code modifications result in instant view/browser updates -- almost comparable to changing CSS/JS in the browser's dev tools directly.

Reply:
Ahh, that's better. Will apply the changes to the files and commit it again.